sql-queries/cleanup_affiliations.sql

```diff
@@ -0,0 +1,98 @@
+DELETE FROM assertions_affiliations
+WHERE affiliation_id IN (
+    SELECT id FROM affiliations WHERE title ~ '^@[^\s]+(\s*[/.-]?\s*@[^\s]+)*$'
+);
+
+DELETE FROM affiliations 
+WHERE title ~ '^@[^\s]+(\s*[/.-]?\s*@[^\s]+)*$';
+
+
+-- Step 2: Delete affiliations with titles that contain only an email address and remove corresponding rows from assertions_affiliations
+DELETE FROM assertions_affiliations
+WHERE affiliation_id IN (
+    '921126d3-3cae-4ba9-8827-da071363c7d6',
+    '6d083246-2d86-4070-a155-949ada1e6ed4',
+    'bb89798e-a740-42c2-b9e4-21581e70137b',
+    '6c465df4-8c61-4198-ad82-40a067a80d32',
+    'd5a9d4b5-901d-4a83-af3a-d8dd8d3cacb8',
+    'be5c7808-ff2a-433d-8d82-b7f3764ef245',
+    '1d62d4f3-4e0e-4c70-8a55-2bad9b10b699',
+    '25071587-3848-47f8-ac52-8c8ac64fdc61',
+    '84776ba1-a058-49dd-95ec-ac4175fa4591',
+    '5cf0f0be-d1cf-4f75-874e-8ab4c41e9587'
+);
+
+DELETE FROM affiliations
+WHERE id IN (
+    '921126d3-3cae-4ba9-8827-da071363c7d6',
+    '6d083246-2d86-4070-a155-949ada1e6ed4',
+    'bb89798e-a740-42c2-b9e4-21581e70137b',
+    '6c465df4-8c61-4198-ad82-40a067a80d32',
+    'd5a9d4b5-901d-4a83-af3a-d8dd8d3cacb8',
+    'be5c7808-ff2a-433d-8d82-b7f3764ef245',
+    '1d62d4f3-4e0e-4c70-8a55-2bad9b10b699',
+    '25071587-3848-47f8-ac52-8c8ac64fdc61',
+    '84776ba1-a058-49dd-95ec-ac4175fa4591',
+    '5cf0f0be-d1cf-4f75-874e-8ab4c41e9587'
+);
+
+
+-- Step 3: Update titles to remove email addresses from titles containing an email along with other text
+UPDATE affiliations
+SET title = regexp_replace(
+                title, 
+                '[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Z|a-z]{2,}', 
+                '', 
+                'g'
+            )
+WHERE title ~ '^(?!.*\b[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Z|a-z]{2,}\b$).*([A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Z|a-z]{2,}).*$';
+
+
+-- Step 4: Update rows where the title contains 'E‐mail:' or 'Electronic address:' to emptry string
+WITH prefix_removed AS (
+    -- Remove specific prefixes from titles
+    SELECT 
+        id,
+        REGEXP_REPLACE(
+            title, 
+            'E‐mail:|Electronic address:|E-mail: |Electronic address|E-mail', 
+            '', 
+            'g'
+        ) AS title_without_prefixes
+    FROM 
+        affiliations
+    WHERE 
+        title ~ 'E‐mail:|Electronic address:|E-mail: |Electronic address|E-mail'
+)
+UPDATE affiliations
+SET title = prefix_removed.title_without_prefixes
+FROM prefix_removed
+WHERE affiliations.id = prefix_removed.id;
+
+
+-- Step 5: Update titles to leading/trailing spaces, and special characters
+WITH cleaned AS (
+    SELECT 
+        id,
+        title,
+        regexp_replace(
+            regexp_replace(
+                trim(both '\n' from trim(both ' ' from title)), -- Trim whitespace/newlines
+                '^[\.,\*;:\s]+|[\.,\*;:\s]+$', '' -- Remove leading/trailing special characters ., *, ;, :, and spaces but keep parentheses
```

Comment:
We do still need to remove _some_ parentheses, but only those where the first and last chars in the title string are `(` and `)`, ex `(University of Kopenhagen)` should become `University of Kopenhagen`

Reply:
Okay, but what we should do the title of type, there are around 403 records which 2 or more parentheses.
- Deutscher Wetterdienst (ZAK) (DWD)
- (1CIBERNED, Spain. 2Cajal Institute (CSIC),)
- (Nederlands Interdisciplinair Demografisch Instituut (NIDI-KNAW)/ Rijksuniversiteit Groningen; Erasmus Universiteit Rotterdam)

Reply:
If the first char of the title value is `(` and the last char is `)` , those 2 paren chars should be removed. Other paren chars should be left as is. Ex, `(Nederlands Interdisciplinair Demografisch Instituut (NIDI-KNAW)/ Rijksuniversiteit Groningen; Erasmus Universiteit Rotterdam)` should become `Nederlands Interdisciplinair Demografisch Instituut (NIDI-KNAW)/ Rijksuniversiteit Groningen; Erasmus Universiteit Rotterdam`.

Reply:
Oh okay, got it. Thanks Liz, Will update the query.